docs/data/genomic_harmonization.rst

```diff
@@ -0,0 +1,238 @@
+- GATK-GenotypGVCF-VQSR
+- Strelka2-somatic-mode
+- Mutect2-somatic-mode
+- Mutect2-tumor-only-mode
+- Vardict-single-sample-mode
+- Vardict-paired-sample-mode
+- Control-FREEC-somatic-mode
+- Control-FREEC-germline-mode
+- StringTie-expression
+- Manta-somatic
+- Manta-germline
+- Lumpy-somatic
+- Lumpy-germline
+- RSEM
+- Kallisto
+- STAR-Fusion
+- Arriba
+- Peddy
+
+
+Relocating Result Files
+-----------------------
+
+WPpelines run in Cavatica have their results output back into
+Cavatica-controlled storage and need to be moved back into Kids First owned
+buckets.
+This occurs when all harmonization tasks have been completed and verified for
+a project by copying the files into the mounted S3 volume and removing the
+original files from Cavatica to save costs.
+The files are copied into prefixes within the mounted study bucket that
+correspond to the pipelines from which the resulted.
+The next section outlines more about the structure of the harmonized files.
+
+
+Study Bucket Organization
+-------------------------
+
+Study buckets use the naming convention: ``kf-study-us-east-1-prd-sd-xxxxxxxx``
+They contain both the original data from the sequencing centers and harmonized
+output data.
+The sequencing center, or ``source``, data is minimally organized and kept
+mostly in the same structure that it was received in.
+The harmonized data is partitioned roughly into categories that are determined
+by what workflow the results came from.
+Below is the desired structure for a study bucket's organization around the
+genomic data:
+
+.. code::
+
+    s3://kf-study-us-east-1-prd-sd-xxxxxxxx/
+    ├── harmonized-data/
+    │   ├── aligned-reads/
+    │   ├── raw-gvcf/
+    │   ├── simple-variants/
+    │   ├── family-variants/
+    │   ├── copy-number-variations/
+    │   ├── structural-variations/
+    │   ├── gene-expressions/
+    │   └── gene-fusions/
+    └── source-data/
+        └── {files_from_SeqCenter}
+
+.. Note::
+   Older studies may store harmonized data in a slightly different structure
+   within the ``harmonized/`` prefix.
+   The above is a newer standard for files within the ``harmonized-data/``
+   prefix.
+
+
+Tagging and Metadata
+--------------------
+
+Cavatica supports tagging and metadata attributes on files which may be
+utilized by Cavatica users for filtering and running tasks.
+This metadata is populated on files by copying additional data about the file
+from the Kids First dataservice after the files have been moved to their final
+location within their Kids First bucket.
+
+
+Delivering Data
+---------------
+
+Data is delivered to the investigator by inviting the investigator's Cavatica
+user to the delivery project under the ``kids-first-drc`` account as a member
+with `copy` permissions.
+The delivery project exposes the harmonized data through files which have been
+mounted from the Kids First study bucket and tagged in Cavatica with relevant
+information from the dataservice.
+
+
+Publicizing Data
```

Comment:
this section is unrelated to the page topic

Reply:
Making the resulting harmonized data public seems to be related to harmonization to me?